backend/controller/sql/schema/20240815164808_async_calls_cron_job_key.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+
+ALTER TABLE cron_jobs
```

Comment:
I suspect the question is yes, but for existing deployments will this break their existing cron jobs? I talked to our customer and they said it's okay for now to keep wiping the database, but at some point we'll need a mechanism for dealing with this.

Reply:
I actually don't think it will break existing cron jobs; we drop the `state` and introduce `last_execution` which is what's used for scheduling. So, in the worst case, the first job per cron after the migration would run once more than intended

Reply:
Aaah, nice nice.